pineappl/src/grid.rs

```diff
@@ -758,6 +804,93 @@ impl Grid {
         Ok(())
     }
 
+    /// Return a vector containing the type of convolutions performed with this grid.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the metadata key--value pairs `convolution_particle_1` and `convolution_type_1`,
+    /// or `convolution_particle_2` and `convolution_type_2` are not correctly set.
+    #[must_use]
+    pub fn convolutions(&self) -> Vec<Convolution> {
+        self.key_values().map_or_else(
+            // if there isn't any metadata, we assume that proton-PDFs are used
```

Comment:
-> `that two unpolarized proton PDFs are used`

Reply:
Changed in commit ba94ab9cf1a300b7727c57ed4d05a87b86788f93.